### PR TITLE
add words on Enumeration range formatting

### LINKED ISSUE #19: Formatting of values in Ranges
There are different ways of formatting values in ranges:

```
    _type.contents                Real
    _enumeration.range            0.0:1.0
```
```
    _type.contents                Real
    _enumeration.range            0.:180.
```
```
    _type.contents                Real
    _enumeration.range            0:1.0
```
```
    _type.contents                Real
    _enumeration.range            0:90
```
```
    _type.contents                Integer
    _enumeration.range            1:48
```
```
    _type.contents                Real
    _enumeration.range            0.0:100.
```

IIRC, CIF2 specification separated the formatting of a value from it's interpretation as an integer/real, so this is purely cosmetic, but...

For consistency, should there be a standard way of doing things?

Reals -> 1.0
Integers -> 1

??

### PR DESCRIPTION
**Pull request by rowlesmr**
Will close #19 

Integer ranges have no decimal point. Real ranges have a decimal point and at least one digit after the decimal point.

### CODE REVIEW

File: accepted/ddlm_dictionary_style_guide.md [hunk 1]
```diff
@@ -246,7 +246,28 @@ compound objects.
                                      '_dictionary_audit.version'
                                      '_dictionary_audit.date'
                                      '_dictionary_audit.revision']
+```
+
+### 2.5 Enumeration ranges
+
+When giving enumeration ranges for `Integer` data items, no decimal
+point shall be used. When giving enumeration ranges for `Real` data
```
Comment (vaitkus) [inline accepted/ddlm_dictionary_style_guide.md:254]:
```suggestion
point should be used. When giving enumeration ranges for `Real` data
```

File: accepted/ddlm_dictionary_style_guide.md [hunk 2]
```diff
@@ -246,7 +246,28 @@ compound objects.
...
+items, a decimal point shall be used with at least one digit after
```
Comment (vaitkus) [inline accepted/ddlm_dictionary_style_guide.md:255]:
```suggestion
items, a decimal point should be used with at least one digit after
```

File: accepted/ddlm_dictionary_style_guide.md [hunk 3]
```diff
@@ -246,7 +246,28 @@ compound objects.
...
+the point. Any following digits, if present, must be non-zero.
 
+#### Examples
+
+```
+# Integers
```
Comment (vaitkus) [inline accepted/ddlm_dictionary_style_guide.md:261]:
```suggestion
# Integer ranges
```

File: accepted/ddlm_dictionary_style_guide.md [hunk 4]
```diff
@@ -246,7 +246,28 @@ compound objects.
...
+1:230
+0:
+-8:8
+
+# Reals
```
Comment (vaitkus) [inline accepted/ddlm_dictionary_style_guide.md:266]:
```suggestion
# Real number ranges
```

**Review by vaitkus — COMMENTED** [review]
I suggested minor wording changes to make the suggestion more similar to the rest of the text.

Also, the version, changelog and other metadata items in the text should be updated.

As a side note, it would be best if in the future the changes in the PR would not include additional formatting changes such as white space removal as it makes the PR harder to review. I replicated the whitespace changes in the trunk, so they no longer appear in the PR diff.

**Comment by jamesrhester** [comment]
Looks good, I think we need to bump the minor version and date, and add to the Changelog at the bottom. We also need to update for new DDL attributes which we could do in the same version.

**Pull request merged**